ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -426,12 +426,16 @@ public boolean maybeFailedDueToOfflineReplicas() {
     }
     if (hasFailedOnCrossColoNotFound() && allReplicaCount - totalNotFoundCount >= replicaSuccessTarget
         && totalOfflineReplicaCount > 0) {
+      Map<ReplicaState, List<ReplicaId>> replicasByState = getReplicasByState(null,
+          EnumSet.of(ReplicaState.BOOTSTRAP, ReplicaState.STANDBY, ReplicaState.LEADER, ReplicaState.INACTIVE,
```

Comment:
can u put this enum in a common file so that u and others can reuse ?

Reply:
1. About the message, good point. I've removed the printout.
2. Will use one existing variable. So the line will be removed.